src/fastoad_cs25/models/geometry/geom_components/wing/components/compute_center_chord.py

```diff
@@ -0,0 +1,93 @@
+"""
+Estimation of center wing chord (l1')
+"""
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2025 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import openmdao.api as om
+from scipy.interpolate import interp1d
+
+
+class ComputeCenterChord(om.ExplicitComponent):
```

Comment:
I'm noticing it now, but all geometrical components linked with the VTP and HTP have a a `HT` or `VT` in their names but it's not the case for wing. Although I dont' think it would be required to make a change about that now, it's worth keeping in mind.

Reply:
I was thinking about `ComputeWingCenterChord`, as for the VT it is called `ComputeVTCenterChord `I believe, but again no need to do it now